notebooks/STIS/low_count_uncertainties/Low_Count_Uncertainties.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    raw_file = os.path.join('mastDownload', 'HST', target_id, f'{target_id}_tag.fits')
</pre><p>This raw file is also pointing to the same tag_file. Please check the previous comment and also this for the correct implementation. :)</p>

<pre class="ql-syntax" spellcheck="false">raw_file = os.path.join('mastDownload', 'HST', target_id, f'{target_id}_raw.fits')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>